docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,191 @@
+2. You can now enable Renovate and will receive a Pull Request to add a configuration file, **read this first PR carefully** as it will explain what kind of PRs you are going to receive and when.
+3. Make sure to **pick a schedule**, otherwise you will receive PRs at any time of the day. Our team schedules all third party dependencies on the weekend, automatically merging when the tests pass and we investigate the failing ones on monday morning.
+4. **Group Pull Requests**, if every PR has to go through CI, it can become quite heavy, once most PRs succeed, you can start grouping minor and patch updates so that you get a single PR per repository. Investigating issues becomes a bit trickier because of that. To give you some numbers out of 90 repositories enabled with Renovate, we have on average 4 PRs to investigate per week. Everything else is automatically merged.
+5. **Enable auto merging after some time**, make sure your tests are solid and that you won’t upgrade something that wasn’t tested and will break once in production.
+
+## Is all this effort worth it ?
+
+The short answer is yes !
+
+It took us almost a year, way longer than we thought, to catch up with the latest version of everything in our technical stack. But once we were confident enough in our tests to enable automatic merging of pull-requests we were statisfied of the work we did to update our software stack. We knew we would be ready when an unplanned change arrives.
+
+When that day came, in december 2021 with Log4Shell, it was a matter of hours to release the freshly merged Pull Requests, deploy the few applications we had, and notify the teams depending on our libraries. We were so quick to do it, in fact, that we had to do that three times with the chain of vulnerabilities that was found in log4j that week.
+
+Keep in mind though that keeping your dependencies up-to-date is not just about the tooling ([Renovate](https://www.mend.io/free-developer-tools/renovate/), [Dependabot](https://github.com/features/security), [Snyk](https://snyk.io/product/open-source-security-management/), …), you will find a tool that fits your use case.
+
+It’s also about having a process:
+
+- When will you merge this PR?
+- How will you handle the PR that doesn’t build?
+- The new major version of an external library that’s not yet compatible with the rest of your libraries?
+- When will you release this constant flow of library updates?
+- Do you want PRs during the day? during the night? on weekends only?
+
+We know the answers for our situation, we’ll let you decide what the answers are for you :)
```

Comment:
```suggestion
We know the answers for our situation, we’ll let you decide what the answers are for you. :smile:
```

- Use full stop to end the sentence.
- Our docs publishing site supports emojis, so we should be able to use `:smile` here 😉